src/components/FButton.vue

```diff
@@ -8,6 +8,8 @@ component.FButton(
   @blur="emit('blur', $event)"
   @click="handleClick($event)"
   @focus="emit('focus', $event)"
+  @keydown.space.stop="handleClick($event)"
+  @keydown.enter.stop="handleClick($event)"
```

Comment:
This is actually not a click + you have this function definition:
`function handleClick(e: MouseEvent):` 

We won't send a MouseEvent in this case.
I suggest to rename the fn to `handleTrigger`,  and accept a MouseEvent or KeyboardEvent